Maestro/subscriptions.json

```diff
@@ -6,6 +6,11 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 8550
     },
+    "aspnet-dependency-update": {
+      "vsoInstance": "devdiv.visualstudio.com",
+      "vsoProject": "AspNet-Universe-MaestroBot-Update",
```

Comment:
Is this a valid VSTS project? The def looks like it's in `DevDiv` to me: https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=8517